docs/product/insights/mobile/mobile-vitals/index.mdx

```diff
@@ -0,0 +1,87 @@
+- [React Native](/platforms/react-native/tracing/instrumentation/automatic-instrumentation/#slow-and-frozen-frames)
+
+## Frames Delay
+
+Frames Delay is the user-perceived total delayed duration of rendered frames. For example, at a refresh rate of 60fps, if a two frame renders takes 20ms each, the frame delay is 8ms (2 * (20ms - 16ms) = 8ms).
+
+You can track frames delay on the following platforms:
+- [Android](/platforms/android/tracing/instrumentation/perf-v2/#frames-delay)
+- [Flutter](/platforms/flutter/integrations/slow-and-frozen-frames-instrumentation/)
+- [iOS, as part of slow and frozen frames](/platforms/apple/guides/ios/tracing/instrumentation/automatic-instrumentation/#slow-and-frozen-frames)
+
+## Time to Initial Display and Time to Full Display
+
+To track how long it takes a screen to render its first frame and then how long it takes to render its first frame with all content, Sentry measures time to initial display and time to full display, respectively.
+
+- **Time to initial display**: tracks how long it takes your app to render its first frame. This includes app start time on the first screen loaded. It doesn't include any content loaded lazily after the first frame is drawn. Time to initial display is measured automatically and enabled by default.
+
+- **Time to full display**: tracks how long it takes for your mobile application to render its first frame with full content. This includes content loaded asynchronously after the first frame, for example, after loading content from the network. Time to full display is opt-in and requires you to manually call the API to report that the screen has loaded all of its content and is fully displayed.
+
+In the example below, the detail view of the transaction displays the time-to-initial-display span in an Android application:
+
+![The event detail of a transaction with time to initial display and time to full display spans.](./img/time-to-initial-full-display.png)
+
+You can track time to initial display for:
+
+- [Android](/platforms/android/tracing/instrumentation/automatic-instrumentation/#time-to-initial-display)
+- [Apple](/platforms/apple/tracing/instrumentation/automatic-instrumentation/#time-to-initial-display)
+- [Flutter](/platforms/flutter/integrations/routing-instrumentation/#time-to-initial-display)
+- [React Native](/platforms/react-native/performance/instrumentation/time-to-display/#automatic-time-to-initial-display-for-react-navigation)
+
+You can track time to full display for:
+
+- [Android](/platforms/android/tracing/instrumentation/automatic-instrumentation/#time-to-full-display)
+- [Apple](/platforms/apple/tracing/instrumentation/automatic-instrumentation/#time-to-full-display)
+- [Flutter](/platforms/flutter/integrations/routing-instrumentation/#time-to-full-display)
+- [React Native](/platforms/react-native/performance/instrumentation/time-to-display/#time-to-full-display)
```

Comment:
`l`: above when we list the supported platforms, we use `iOS` instead of `Apple` and since we sort them alphabetically, the ordering is different. I prefer `Apple` and the ordering used here